client/settings/advanced-settings-section/experimental-features.js

```diff
@@ -93,17 +93,17 @@ const ExperimentalFeatures = () => {
 	return (
 		<>
 			<h4 ref={ headingRef } tabIndex="-1">
-				{ __( 'Experimental features', 'woocommerce-gateway-stripe' ) }
+				{ __( 'New', 'woocommerce-gateway-stripe' ) }
 			</h4>
 			<CheckboxControl
 				data-testid="new-checkout-experience-checkbox"
 				label={ __(
-					'Try the new checkout experience (early access)',
+					'Try the updated checkout experience',
```

Comment:
I assume the "Try" is because it was an experimental feature. Now we're tweaking this, I think we can use "Enable the updated...".

Reply:
updated. thank you for the suggestions @cesarcosta99 .